it_hardware/demo/purchase_order_confirm.xml

```diff
@@ -69,9 +65,6 @@
         <value model="account.move" eval="obj().env.ref('it_hardware.purchase_order_2').invoice_ids.ids"/>
         <value eval="{'invoice_date': DateTime.today().date()}"/>
```

Comment:
I think you can remove the invoice_date on purchase order invoices as well since you don't post it anymore.